string.c

```diff
@@ -0,0 +1,30 @@
+/*Sean Kee*/
+/*Takes in string input, returns frequency of each character*/
```

Comment:
hmm, your code correctly counts the frequency of the letters of the string, but correctly increments the counts for characters that were never used!